
def call(def exception) {
    echo exception

    echoBanner("WE'RE IN THE FAIL SCRIPT: ${exception?.message} !!!!!!")
}